content/docs/command-reference/exp/show.md

```diff
@@ -147,109 +119,147 @@ metric or param.
+│ └── d384680 [exp-bc055] │ 08:03 PM     │  0.51799 │ 0.92333 │ 500                    │
+└─────────────────────────┴──────────────┴──────────┴─────────┴────────────────────────┘
 ```
 
-You can also filter out any metrics and parameters that do not change across the
-shown experiments:
+You can also use `--drop` to filter columns matching a specific [regex][regex]:
 
 ```dvc
-$ dvc exp show --only-changed
+$ dvc exp show --drop prepare
 ```
 
 ```dvctable
-┏━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━┓
-┃ neutral:**Experiment**            ┃ neutral:**Created**      ┃     metric:**auc** ┃ param:**featurize.max_features** ┃
-┡━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━┩
-│ workspace             │ -            │ 0.61314 │ 1500                   │
-│ 10-bigrams-experiment │ Jun 20, 2020 │ 0.61314 │ 1500                   │
-│ ├── exp-e6c97         │ Oct 21, 2020 │ 0.61314 │ 1500                   │
-│ ├── exp-1dad0         │ Oct 09, 2020 │ 0.57756 │ 2000                   │
-│ └── exp-1df77         │ Oct 09, 2020 │ 0.51676 │ 500                    │
-└───────────────────────┴──────────────┴─────────┴────────────────────────┘
+┏━━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━┓
+┃ neutral:**Experiment**              ┃ neutral:**Created**      ┃ metric:**avg_prec** ┃ metric:**roc_auc** ┃ param:**featurize.max_features** ┃ param:**featurize.ngrams** ┃ param:**train.seed** ┃ param:**train.n_est** ┃ param:**train.min_split** ┃
+┡━━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━┩
+│ workspace               │ -            │  0.60405 │  0.9608 │ 3000                   │ 2                │ 20170428   │ 100         │ 64              │
+│ master                  │ May 29, 2021 │  0.60405 │  0.9608 │ 3000                   │ 2                │ 20170428   │ 100         │ 64              │
+│ ├── 6b338f8 [exp-3315b] │ 08:03 PM     │  0.58589 │   0.945 │ 2000                   │ 2                │ 20170428   │ 100         │ 64              │
+│ ├── d384680 [exp-bc055] │ 08:03 PM     │  0.51799 │ 0.92333 │ 500                    │ 2                │ 20170428   │ 100         │ 64              │
+│ └── d7fdde2 [exp-1b262] │ 08:03 PM     │  0.56447 │ 0.94713 │ 1500                   │ 2                │ 20170428   │ 100         │ 64              │
+└─────────────────────────┴──────────────┴──────────┴─────────┴────────────────────────┴──────────────────┴────────────┴─────────────┴─────────────────┘
 ```
 
-Sort experiments by the `auc` metric, in ascending order:
+You can use `|` in order to remove a list of columns:
 
 ```dvc
-$ dvc exp show --include-params=featurize --sort-by=auc --sort-order=asc
+$ dvc exp show --drop 'avg_prec|train.min_split'
```

Comment:
Hmmmm. Do we recommend this over several `--drop`s? Not sure. Feels like we're documenting regular expressions here. We could skip this one (but maybe add another `--drop` to the previous one).

Reply:
I've removed support for multiple `--drop` in core.

I've considered that joining using `|`  was an easy to teach replacement for either multiple `--drop` or comma-separated lists.

Do you consider that we should still support multiple values?

Reply:
> tend to think contrary to @iesahin. Examples in the ref. should serve as quick ways for users to grasp how the tool does its job and cover some main use cases. In the UG it's unclear we want separate example sections, as we can embed the examples throughout it. I believe @shcheklein also favors this view.

I believe REF examples should be like [tldr](https://tldr.sh/) pages, and without much explanation. People (like me) usually refer to REF to find the exact option quickly.

For UG, I tend to have more explanations, describing "why" and "how" rather verbosely. These may be separated from the UG, though I believe having UG around workflows (rather than commands) might make more sense. In this case, we'll have larger examples.

Reply:
How about something like, "You can use regex to match columns. For example, to remove multiple columns:"